src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+
+    private static String[] colors = {"RED", "ORANGE", "GREEN", "GRAY", "YELLOW"};
```

Comment:
```suggestion
public class ColorSupplier {
    private static final String[] COLORS = {"RED", "ORANGE", "GREEN", "GRAY", "YELLOW"};
```
let's make this a constant field